sdk/servicebus/Azure.Messaging.ServiceBus/tests/Samples/Sample06_Transactions.cs

```diff
@@ -77,42 +77,40 @@ await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitionin
         [Test]
         public async Task TransactionGroup()
         {
-            await using var client = CreateClient();
+            await using var client = new ServiceBusClient(
+                TestEnvironment.ServiceBusConnectionString,
+                new ServiceBusClientOptions
+                {
+                    EnableCrossEntityTransactions = true
+                });
             await using var queueA = await ServiceBusScope.CreateWithQueue(enablePartitioning: false, enableSession: false);
             await using var queueB = await ServiceBusScope.CreateWithQueue(enablePartitioning: false, enableSession: false);
             await using var topicC = await ServiceBusScope.CreateWithTopic(enablePartitioning: false, enableSession: false);
 
-            #region Snippet:ServiceBusTransactionGroup
-
-            // The first sender won't be part of our transaction group.
+            // The first sender won't be part of our transaction and is
```

Comment:
From a client/user perspective would we now suggest to have a dedicated Servicebus Client to make it explicit what participates and what not? The fact that you had to make a comment here seems something is non intuitive in this API and requires further explanation of the runtime behaviour for our future self. How will the users of the SDK understand and deal with this?

Reply:
Yes, with the current design you would need a dedicated client for all entities that should participate in cross-entity txns. I think the fact that the option is on the client options implies that you would need to use the same client for all entities participating. What is less clear is that you may need a separate client if you will be performing operations on a different entity in which send-via wouldn't make sense. I think we will have to rely on documentation and on improving the exception message that occurs in this case.